yarn-project/end-to-end/src/e2e_cross_chain_messaging/cross_chain_messaging_test.ts

```diff
@@ -118,6 +118,23 @@ export class CrossChainMessagingTest {
 
         const { publicClient, walletClient } = createL1Clients(this.aztecNodeConfig.l1RpcUrls, MNEMONIC);
 
+        const underlyingERC20Address = await deployL1Contract(
```

Comment:
Having this be in the CrossChainTestHarness and having the underlyingERC20Address passed to the CrossChainTestHarness.new function just sometimes made it all hard to debug and quite ugly so I decided to just move it here.